### PR TITLE
Feature/df 120 redirects landings

### PR DESCRIPTION
**Pull request by ganqqwerty**
redirection component to the landing pages

### CODE REVIEW

File: alv-portal-ui/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,7 +1,46 @@
...
+        component: PvaLandingPageComponent
+      },
+      {
+        path: 'company',
+        component: PvaLandingPageComponent //right now we provide the same page for pva and the company
+      }
+    ]
+  },
+  {
+    path: 'nza',
```
Comment (birom) [inline alv-portal-ui/src/app/app-routing.module.ts]: What is nza?

File: alv-portal-ui/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,7 +1,46 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { LandingPageComponent } from './core/landing/landing-page/landing-page.component';
+import { JobSeekerLandingPageComponent } from './core/landing/job-seeker-landing-page/job-seeker-landing-page.component';
+import { PvaLandingPageComponent } from './core/landing/pva-landing-page/pva-landing-page.component';
+import { NzaPageComponent } from './core/nza-page/nza-page.component';
+import { HomeComponent } from './core/home/home.component';
```
Comment (birom) [inline alv-portal-ui/src/app/app-routing.module.ts]: I do not find the ./core/home directory.

File: alv-portal-ui/src/app/core/header/header.component.ts [hunk 3]
```diff
@@ -51,4 +51,7 @@ export class HeaderComponent extends AbstractSubscriber implements OnInit {
     }
   }
 
+  logout() {
+    this.authenticationService.logout();
```
Comment (birom) [inline alv-portal-ui/src/app/core/header/header.component.ts]: What happesn with the ``this.user`` after logout?

File: alv-portal-ui/src/app/app-routing.module.ts (hunk 2 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/app-routing.module.ts]: thanks! I didn't add it into the code. fixed

File: alv-portal-ui/src/app/core/header/header.component.ts (hunk 3 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/core/header/header.component.ts]: removed, it's an artifact

File: alv-portal-ui/src/app/app.component.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import { Component, OnInit } from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
 import { AuthenticationService } from './core/authentication/authentication.service';
-import { takeUntil } from 'rxjs/operators';
+import { LandingNavigationService } from './core/landing-navigation.service';
 
 @Component({
   selector: 'os-root',
```
[the remark this answers is not in the document]
Comment (ganqqwerty) [reply alv-portal-ui/src/app/app.component.ts]: will be done in a separate task

File: alv-portal-ui/src/app/app-routing.module.ts (hunk 1 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/app-routing.module.ts]: the page where you choose whether you're company or pva or job-seeker

File: alv-portal-ui/src/app/app.component.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (birom) [reply alv-portal-ui/src/app/app.component.ts]: ok, thx

File: alv-portal-ui/src/app/app-routing.module.ts (hunk 1 above)
Comment (birom) [reply alv-portal-ui/src/app/app-routing.module.ts]: Please rename it to complete-registration.

Comment (ganqqwerty) [reply alv-portal-ui/src/app/app-routing.module.ts]: done

File: alv-portal-ui/src/app/app.component.ts [hunk 5]
```diff
@@ -11,7 +11,9 @@ import { takeUntil } from 'rxjs/operators';
 export class AppComponent implements OnInit {
 
   constructor(translate: TranslateService,
-              private authenticationService: AuthenticationService) {
+              private authenticationService: AuthenticationService,
+              private landingNavigationService: LandingNavigationService
```
Comment (birom) [inline alv-portal-ui/src/app/app.component.ts]: Please remove it is not used.

File: alv-portal-ui/src/app/app-routing.module.ts (hunk 1 above)
Comment (birom) [reply alv-portal-ui/src/app/app-routing.module.ts]: I still see the ``nza`` path.  Have you pushed it?

**Pull request merged**